db/fixtures/ae_datastore/ManageIQ/System/CommonMethods/QuotaMethods.class/__methods__/limits.rb

```diff
@@ -42,12 +42,21 @@ def vmdb_object(object, id)
 def parent_model_value(attr)
   value = $evm.parent[attr].to_i
   $evm.log(:info, "Quota Model #{attr}: #{value}") unless value.zero?
+  if attr.ends_with?("storage") || attr.ends_with?("memory")
+    value *= 1024**2
```

Comment:
@mkanoor 
The calculation looks odd because we're compensating for tag value differences as we discussed a few weeks ago. 

We ship with quota categories and tags. The description for both max_quota_memory and max_quota_storage state (GB), but the tag values differ, for example.
The max_quota_memory category, tag value for 10GB is 10240.   
The max_quota_storage category, tag value for 10GB is 10.

This is one of the issues to be discussed for the next release. Since we don't want to change the tag values for existing customers, I was hoping we'd add a new set of quota category/tags with the proper values.